src/main/java/org/jcodec/containers/mp4/boxes/UdtaBox.java

```diff
@@ -35,12 +37,45 @@ public Box newBox(Header header) {
     public UdtaBox(Header atom) {
         super(atom);
     }
-    
+
     public MetaBox meta() {
         return NodeBox.findFirst(this, MetaBox.class, MetaBox.fourcc());
     }
 
     public static String fourcc() {
         return FOURCC;
     }
+
+    public String latlng() {
+        Box gps = findGps(this);
+        if (gps == null) return null;
+        ByteBuffer data = getData(gps);
+        if (data == null) return null;
+        if (data.remaining() < 4) return null;
+        data.getInt(); //skip 4 bytes
+        byte[] coordsBytes = new byte[data.remaining()];
+        data.get(coordsBytes);
+        String latlng = new String(coordsBytes);
```

Comment:
This is bad code, as converting `byte[]` to a String ALWAYS requires to specify a character set. This use implicit default given by the runtime, which is NOT the character set of the metadata. Please consider using forbiddenapis usage in jcodec to detect such problematic code like missing charsets, implicit default locales on lower/uppercasing,....